plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtDomainXMLParser.java

```diff
@@ -343,7 +356,8 @@ public boolean parseDomainXML(String domXML) {
 
                 watchDogDefs.add(def);
             }
-
+            extractCpuTuneDef(rootElement);
+            extractCpuModeDef(rootElement);
```

Comment:
I'd probably prefer these methods returning their respective results and thos being assigned here, but this is way better than the inline parising happening above :+1:

Reply:
done